metadata-ingestion/src/datahub/ingestion/source/usage/redshift_usage.py

```diff
@@ -223,7 +223,12 @@ def _get_redshift_history(
                 event_dict["endtime"] = event_dict.get("endtime").__str__()
 
             logger.debug(f"event_dict: {event_dict}")
-            events.append(event_dict)
+            # filter based on schema and table pattern
+            if self.config.schema_pattern.allowed(event_dict['schema'])\
```

Comment:
any plans to include `view_pattern` as well?

Reply:
@rslanka, it seems like the script wasn't ingesting view related usage stats in the first place. I've looked at the code and couldn't find anything. Did a small test run as well; view usage didn't get ingested.